Backend/app/rider-platform/rider-app/Main/src-read-only/API/Action/Dashboard/RideBooking/Booking.hs

```diff
@@ -0,0 +1,34 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-unused-imports #-}
+
+module API.Action.Dashboard.RideBooking.Booking
+  ( API.Types.Dashboard.RideBooking.Booking.API,
+    handler,
+  )
+where
+
+import qualified API.Types.Dashboard.RideBooking.Booking
+import qualified Domain.Action.Dashboard.RideBooking.Booking as Domain.Action.Dashboard.RideBooking.Booking
+import qualified "this" Domain.Action.UI.Booking
+import qualified "this" Domain.Types.Booking
+import qualified "this" Domain.Types.Booking.API
+import qualified Domain.Types.Merchant
+import qualified "this" Domain.Types.Person
+import qualified Environment
+import EulerHS.Prelude
+import qualified EulerHS.Prelude
+import qualified Kernel.Prelude
+import qualified Kernel.Types.Beckn.Context
+import qualified Kernel.Types.Id
+import Kernel.Utils.Common
+import Servant
+import Tools.Auth
+
+handler :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> Environment.FlowServer API.Types.Dashboard.RideBooking.Booking.API)
+handler merchantId city = postBookingStatus merchantId city :<|> getBookingList merchantId city
+
+postBookingStatus :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> Kernel.Types.Id.Id Domain.Types.Booking.Booking -> Kernel.Types.Id.Id Domain.Types.Person.Person -> Environment.FlowHandler Domain.Types.Booking.API.BookingAPIEntity)
+postBookingStatus a4 a3 a2 a1 = withDashboardFlowHandlerAPI $ Domain.Action.Dashboard.RideBooking.Booking.postBookingStatus a4 a3 a2 a1
```

Comment:
earlier it was:

```
callBookingStatus :: ShortId DM.Merchant -> Id SRB.Booking -> Id DP.Person -> FlowHandler DB.BookingAPIEntity
callBookingStatus merchantId bookingId personId = do
  m <- withDashboardFlowHandlerAPI $ findMerchantByShortId merchantId
  UB.bookingStatus bookingId (personId, m.id)

bookingStatus :: Id SRB.Booking -> (Id Person.Person, Id Merchant.Merchant) -> FlowHandler BookingAPIEntity
bookingStatus bookingId = withFlowHandlerAPI . DBooking.bookingStatus bookingId
```
so `withFlowHandlerAPI` wrapper changed to `withDashboardFlowHandlerAPI` in all `RideBooking` apis